CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to MDN Documentation Linter rules
+
+# How to install
+The code is not yet proposed as an npm package, you need to manually clone the repository
+`git clone git@github.com:mdn/doc-linter-rules.git`
```

Comment:
I suggest using triple quoting so that these commands are on their own lines, like:

The code is not yet proposed as an npm package, you need to manually clone the repository

```
git clone git@github.com:mdn/doc-linter-rules.git
```

Then you have to install the dependencies using npm